### PR TITLE
Pal 131 move documentation to new repo

### PR DESCRIPTION
**Pull request by m78233**
Initial review was done on the README.  The changes for that have been started.

### CODE REVIEW

**Comment by m78233** [comment]
Initial feedback includes:
1) change the strap line to “A tool for complex and scalable data access policy enforcement”
2)Highlight for users at first glance what the USP is. So either through bold words or headings
3) Queries should be a github issue on the project and use the question tag.
4) Sentence starting with “When a user submits a query" needs re-wording
5) Change name of file to Developer Guide
6) Current version is 0.4.0
7) Double-check that there is still an issue with Windows build
8) Re-word description of each of the repos to the list given
9) Move the description to the top or a separate page

File: book.json [hunk 1]
```diff
@@ -1,34 +0,0 @@
-{
```
Comment (ac74475) [inline book.json:1]: If we choose to continue building the documentation using gitbooks then this is probably required

File: cd/deploy.sh [hunk 2]
```diff
@@ -1,89 +0,0 @@
...
-        git commit -a -m "prepare release $artifactId-$RELEASE_VERSION"
-        git tag $artifactId-$RELEASE_VERSION
-        git push origin --tags
-        git push
-
-        echo ""
-        echo "--------------------------------------"
-        echo "Updating doc"
-        echo "--------------------------------------"
-        npm install -g gitbook-cli > /dev/null 2>&1
```
Comment (ac74475) [inline cd/deploy.sh:53]: Might want to keep this part of the script for building the gitbook pages

File: contents.md [hunk 3]
```diff
@@ -1,76 +0,0 @@
-# Summary
```
Comment (ac74475) [inline contents.md:1]: This file would be required for the gitbooks stuff, but could be slimmed down a lot

File: doc/gitbook.md [hunk 4]
```diff
@@ -1,24 +0,0 @@
-# Building the documentation
```
Comment (ac74475) [inline doc/gitbook.md:1]: Again useful for using gitbooks

File: doc/scripts/buildGitbook.sh [hunk 5]
```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
-# This should be run from the root of the project: ./doc/scripts/buildGitbook.sh
```
Comment (ac74475) [inline doc/scripts/buildGitbook.sh:3]: useful for building the gitbook stuff, can remove the java doc stuff though

File: doc/scripts/serveGitbook.sh [hunk 6]
```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
```
Comment (ac74475) [inline doc/scripts/serveGitbook.sh:1]: useful for local testing of gitbook build

**Comment by m78233** [comment]
Initial feedback, item 1 and 2
The strap line has been changed to "A Tool for Complex and Scalable Data Access Policy Enforcement" and the font larger header, h3.  As it is, I do think that this sentence is a good USP.  I thought of a couple of permutations that could also be possible candidates:  "A Tool for Managing Data Access Policy"; "A Unique Tool for Managing your Corporate Data Access Policy"; and with a different spin, "Palisade Aim is to Provide a Stable Framework for Defining and Enforcing Data Access Policies."  Personally my vote still for the first one.

**Comment by m78233** [comment]
2) Queries should be a github issue on the project and use the question tag.
First crack at it with a brief description on what to do, that we will reply using the same and a link.

**Comment by m78233** [comment]
3 Sentence starting with “When a user submits a query" needs re-wording

Read it a dozen times, and concluded it was a sin of copy and paste.  The paragraph did not belong.  I have taken some of the text from your comments and added them to the previous paragraph, and then removed the “When …” paragraph.  A lot of it was redundant to what had already been said.  The exception was the concept of a data local process for optimum performance needs to either be made into a paragraph dedicated to this one topic or kept out of the README.  The reasoning is that it is more of a technical detail that will be covered in the developer’s guide.  I have leaned towards the later as I think it is an easier read.

**Comment by m78233** [comment]
Change name of file to Developer Guide
Moved the information into the Developer Guide

**Comment by m78233** [comment]
Current version is 0.4.0
Done

**Comment by m78233** [comment]
Double-check that there is still an issue with Windows build

It reads like this is no longer an issue.  I have removed the comment from the text, but I will still double check with the Window users on the team.

**Comment by m78233** [comment]
Re-word description of each of the repos to the list given
Done

**Comment by m78233** [comment]
Move the description to the top or a separate page

Moved the more technical text out of the README.m and put it in the developer_guide.md

File: book.json (hunk 1 above)
Comment (m78233) [reply book.json:1]: Put it back in.

File: cd/deploy.sh (hunk 2 above)
Comment (m78233) [reply cd/deploy.sh:53]: This is a script that included the building of the gitbook along with other features.  Now what we have is the scripts for gitbook as a standalone set. I don't think that we will need the deploy.sh any longer

File: contents.md (hunk 3 above)
Comment (m78233) [reply contents.md:1]: Rebuilt (and renamed back to the default name) the summary.md

File: doc/gitbook.md (hunk 4 above)
Comment (m78233) [reply doc/gitbook.md:1]: Put back in.

File: doc/scripts/buildGitbook.sh (hunk 5 above)
Comment (m78233) [reply doc/scripts/buildGitbook.sh:3]: Put it back in.

**Pull request merged**